src/components/Tabs/Tab/Tab.tsx

```diff
@@ -1,14 +1,42 @@
 /* eslint-disable jsx-a11y/anchor-is-valid,jsx-a11y/click-events-have-key-events */
-import React, { useRef, forwardRef } from "react";
+import React, { FC, forwardRef, ReactElement, useRef } from "react";
 import NOOP from "lodash/noop";
-import PropTypes from "prop-types";
 import cx from "classnames";
 import useMergeRefs from "../../../hooks/useMergeRefs";
-import Icon from "../../Icon/Icon";
+import Icon, { IconSubComponentProps } from "../../Icon/Icon";
```

Comment:
[Future changes]
It not merged yet but for future we need to change `IconSubComponentProps` according to:
https://github.com/mondaycom/monday-ui-react-core/pull/956